app/assets/javascripts/components/locales/oc.jsx

```diff
@@ -1,128 +1,177 @@
+ * Thanks!
+ */
 const oc = {
-  "column_back_button.label": "Tornar",
-  "lightbox.close": "Tampar",
-  "loading_indicator.label": "Cargament…",
-  "status.mention": "Mencionar",
-  "status.delete": "Escafar",
-  "status.reply": "Respondre",
-  "status.reblog": "Partejar",
-  "status.favourite": "Apondre als favorits",
-  "status.reblogged_by": "{name} a partejat :",
-  "status.sensitive_warning": "Contengut embarrassant",
-  "status.sensitive_toggle": "Clicar per mostrar",
-  "status.show_more": "Desplegar",
-  "status.show_less": "Tornar plegar",
-  "status.open": "Desplegar aqueste estatut",
-  "status.report": "Senhalar @{name}",
-  "status.load_more": "Cargar mai",
-  "status.media_hidden": "Mèdia rescondut",
-  "video_player.toggle_sound": "Activar/Desactivar lo son",
-  "video_player.toggle_visible": "Mostrar/Rescondre la vidèo",
-  "account.mention": "Mencionar",
-  "account.edit_profile": "Modificar lo perfil",
-  "account.unblock": "Desblocar",
-  "account.unfollow": "Quitar de sègre",
   "account.block": "Blocar",
-  "account.mute": "Rescondre",
-  "account.unmute": "Quitar de rescondre",
+  "account.disclaimer": "Aqueste compte es sus una autra instància. Los nombres pòdon èsser mai grandes.",
+  "account.edit_profile": "Modificar lo perfil",
   "account.follow": "Sègre",
-  "account.posts": "Estatuts",
-  "account.follows": "Abonaments",
   "account.followers": "Abonats",
   "account.follows_you": "Vos sèc",
+  "account.follows": "Abonaments",
+  "account.mention": "Mencionar @{name}",
+  "account.mute": "Rescondre @{name}"
+  "account.posts": "Estatuts",
+  "account.report": "Senhalar @{name}",
   "account.requested": "Invitacion mandada",
-  "account.report": "Senhalar",
-  "account.disclaimer": "Aqueste compte es sus una autra instància. Los nombres pòdon èsser mai grandes.",
-  "getting_started.heading": "Per començar",
-  "getting_started.about_addressing": "Podètz sègre los estatuts de qualqu’un en picant son identificant e lo domeni de l’instància separat amb un @ coma una adreça de corrièl dins lo camp de recèrca.",
-  "getting_started.about_shortcuts": "S’aquesta persona emplega la meteissa instància que vos l’identifican basta. Atal foncionan tanben las mencions dins vòstres estatuts.",
-  "getting_started.about_developer": "Per sègre lo desvolopaire d’aqueste projècte : Gargron@mastodon.social",
-  "getting_started.open_source_notice": "Mastodon es un logicial liure. Podètz contribuir e mandar vòstres comentaris e rapòrt de bug via{github} sus GitHub.",
-  "column.home": "Acuèlh",
-  "column.community": "Fil public local",
-  "column.public": "Fil public global",
-  "column.notifications": "Notificacions",
+  "account.unblock": "Desblocar @{name}",
+  "account.unfollow": "Quitar de sègre",
+  "account.unmute": "Quitar de rescondre @{name}",
+  "boost_modal.combo": "Podètz picar {combo} per passar aquò lo cop que ven",
   "column.blocks": "Personas blocadas",
+  "column.community": "Fil public local"
```

Comment:
Lines 20, 29 (shown above) and 167 are all missing the comma at the end of the line.